daemon/main.c

```diff
@@ -789,6 +791,9 @@ static void options(int *argc, char ***argv) {
 	if (rtpe_config.control_tos < 0 || rtpe_config.control_tos > 255)
 		die("Invalid control-ng TOS value");
 
+	if (rtpe_config.max_recv_iters < MAX_RECV_ITERS || rtpe_config.control_tos > MAX_RECV_ITERS*10)
```

Comment:
copy/paste error (left `control_tos`)

I don't think there's a reason to bound-check this value at all, except maybe to make sure it's not negative.

Reply:
I think the value should be at least 1, otherwise the error occurs at the first iteration of the loop.